helm-chart/values.yaml

```diff
@@ -41,9 +43,21 @@ securityContext: {}
   # runAsNonRoot: true
   # runAsUser: 1000
 
+pvc:
+  storageClassName: standard
+  conf:
+    size: 1Gi
+  data:
+    size: 10Gi
+  env:
+    size: 1Gi
+  log:
+    size: 1Gi
+
 service:
   type: ClusterIP
   port: 80
+  targetPort: 80
```

Comment:
If you add a values for the port why not add it for the service as well ?

{{ .Values.container.port }}

Reply:
Hello,

I'm not sure to understand, the port is custom, in the `service.yaml` and in `deployment.yaml`.
Did I forget something ?

Reply:
@finaldzn Can you elaborate?

Reply:
I was wrong and tought for some reason that this was an other file then values.yaml. My bad sorry :)